x/callback/LICENSE

```diff
@@ -0,0 +1,97 @@
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+-----------------------------------------------------------------------------
+
+Business Source License 1.1
+
+Parameters
+
+Licensor:             Archway Services, Ltd.
+
+Licensed Work:        Archway
+
+Change Date:          2026-05-30
```

Comment:
3 years from date of publication:
So we can make it 2027 and the day of merge commit (maybe even 08/01 as we start audits for this module then);